deploy/run.sh

```diff
@@ -12,22 +12,37 @@ if [ ! -x $CADVISOR_PORT ]; then
   EXTRA_ARGS="$EXTRA_ARGS --cadvisor_port=$CADVISOR_PORT"
 fi
 
+# Select what source sink to use. Options: "influxdb,gcm"
+# Default is InfluxDB.
+if [ -x $SINK ]; then
+  SINK="influxdb"
+fi
+
 HEAPSTER="/usr/bin/heapster $EXTRA_ARGS "
-# Check if InfluxDB service is running
-if [ ! -z $KUBERNETES_RO_SERVICE_HOST ]; then
-  # TODO(vishh): add support for passing in user name and password.    
-  INFLUXDB_ADDRESS=""
-  if [ ! -z $MONITORING_INFLUXDB_SERVICE_HOST ]; then
-    INFLUXDB_ADDRESS="${MONITORING_INFLUXDB_SERVICE_HOST}:${MONITORING_INFLUXDB_SERVICE_PORT}"
+
+if [ "$SINK" == "influxdb" ]; then
+  # Check if InfluxDB service is running
```

Comment:
nit: The comment here should be "check if we are running in kubernetes". i see that it was incorrect to begin with.